src/nrc/api/tests/test_scopes.py

```diff
@@ -0,0 +1,361 @@
+from unittest.mock import patch
+from uuid import uuid4
+
+from django.test import override_settings
+
+import requests_mock
+from rest_framework import status
+from rest_framework.test import APITestCase
+from vng_api_common.tests import JWTAuthMixin, get_operation_url
+
+from nrc.api.choices import SequencetypeChoices
+from nrc.datamodel.choices import ProtocolChoices
+from nrc.datamodel.tests.factories import DomainFactory, SubscriptionFactory
+
+from ..scopes import (
+    SCOPE_DOMAINS_CREATE,
+    SCOPE_DOMAINS_READ,
+    SCOPE_EVENTS_PUBLISH,
+    SCOPE_SUBSCRIPTIONS_CREATE,
+    SCOPE_SUBSCRIPTIONS_DELETE,
+    SCOPE_SUBSCRIPTIONS_READ,
+    SCOPE_SUBSCRIPTIONS_UPDATE,
+)
+
+
+@override_settings(
+    LINK_FETCHER="vng_api_common.mocks.link_fetcher_200",
+    ZDS_CLIENT_CLASS="vng_api_common.mocks.MockClient",
+)
+class ScopeSubscriptionsRead(JWTAuthMixin, APITestCase):
+    scopes = [SCOPE_SUBSCRIPTIONS_READ]
+    source = "urn:nld:oin:00000001234567890000:systeem:Zaaksysteem"
+    sink = "https://endpoint.example.com/webhook"
+    data = {
+        "protocol": ProtocolChoices.HTTP,
+        "source": source,
+        "sink": sink,
+        "types": [
+            "Type A",
+            "Type B",
+        ],
+    }
+
+    def test_correct_scope(self):
+        """
+        test /subscriptions GET:
```

Comment:
Deze test heb je heel creatief gemaakt ;) Hier doe je een `GET` naar de URL van de `subscriptions_update` (de `PUT`). Als je de `GET` wil testen zou ik hier de `subscriptions_read` verwachten in je `get_operation_url` aanroep. Ook heb je dan niet de mocks nodig die je gebruikt.